docs/source/Scripts/Common/External/bowtie.rst

```diff
@@ -0,0 +1,19 @@
+=========
+bowtie.py
+=========
+
+.. rubric:: Usage adn command line options
```

Comment:
Typo with `adn` Usage `and` command line options

Reply:
Thanks for catching that